test/e2e_node/device_plugin_test.go

```diff
@@ -610,7 +610,7 @@ func testDevicePlugin(f *framework.Framework, pluginSockDir string) {
 			}, 30*time.Second, framework.Poll).ShouldNot(gomega.HaveOccurred(), "cannot fetch the compute resource assignment after kubelet restart")
 
 			err, _ = checkPodResourcesAssignment(v1PodResources, pod1.Namespace, pod1.Name, pod1.Spec.Containers[0].Name, SampleDeviceResourceName, []string{devID1})
-			framework.ExpectNoError(err, "inconsistent device assignment after pod restart")
+			framework.ExpectNoError(err, fmt.Sprintf("inconsistent device assignment after pod restart, v1PodResources is: %+v, pod1 is: %+v", v1PodResources, pod1))
```

Comment:
Please show how the log looks like after this change. Do we need the same or similar changes in any other cases?

Also would it help to improve `checkPodResourcesAssignment` to include more details on what it looked for and what devices were actually found? It may improve debuggability of many tests at once

Reply:
```
no resources found for default/test/test-1 in listpodresources [default/test1/test1-abc,default/test1/test1-init,]
```

I agree add to `checkPodResourcesAssignment`, It can improve debuggability of many tests at once.